pkgs/top-level/all-packages.nix

```diff
@@ -1,11 +1,17 @@
-/* The top-level package collection of nixpkgs.
- * It is sorted by categories corresponding to the folder names in the /pkgs
- * folder. Inside the categories packages are roughly sorted by alphabet, but
- * strict sorting has been long lost due to merges. Please use the full-text
- * search of your editor. ;)
- * Hint: ### starts category names.
- */
-{ lib, noSysDirs, config, overlays }:
```

Comment:
accidental nixfmt?

Reply:
Likely :sweat_smile:

Reply:
Rebased on latest nixos-unstable and fixed that commit

Reply:
GitHub still shows a merge conflict though?

Reply:
I rebased on master now, and the conflict is resolved